app/dashboard/views.py

```diff
@@ -31,18 +32,22 @@
 
 from app.utils import ellipses, sync_profile
 from dashboard.helpers import normalizeURL, process_bounty_changes, process_bounty_details
-from dashboard.models import Bounty, BountySyncRequest, Interest, Profile, ProfileSerializer, Subscription, Tip
+from dashboard.models import Bounty, BountySyncRequest, Interest, Profile, ProfileSerializer, Subscription, Tip, CoinRedemption, CoinRedemptionRequest
```

Comment:
Once syntax errors are resolved throughout the additions, please run `isort app/dashboard/views.py` from your `web` repository root directory, so the imports reflect something like:

```python
from dashboard.models import (
    Bounty, BountySyncRequest, CoinRedemption, CoinRedemptionRequest, Interest, Profile, ProfileSerializer,
    Subscription, Tip,
)
```
and
`from web3 import HTTPProvider, Web3`